catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/email/EmailNotifier.java

```diff
@@ -53,48 +48,36 @@
 
     private String fromEmail;
 
-    private String mailHost;
-
     private SubscriptionsPersistentStore subscriptionsPersistentStore;
 
+    private SmtpClient smtpClient;
+
     /**
      * The {@code bodyTemplate} and {@code subjectTemplate} may contain the tags supported by the
      * {@code metacardFormatter}.
      *
-     * @param bodyTemplate                 must be non-null
-     * @param subjectTemplate              must be non-null
-     * @param fromEmail                    must be non-null
-     * @param mailHost                     must be non-null
-     * @param metacardFormatter            must be non-null
-     * @param subscriptionsPersistentStore must be non-null
+     * @param bodyTemplate                 A string that represents the email body, will be passed to MetacardFormatter.
```

Comment:
Might be useful to link the MetacardFormatter in the javadoc
`{@link org.codice.ddf.catalog.ui.query.monitor.api.MetacardFormatter}`